bin/mail/mailing-common/mailing-list-utils.rb

```diff
@@ -189,3 +189,18 @@ def query_all_emails_at_domain(domain)
     ALL.merge! results
   end
 end
+
+# Accepts a hash of SOLR queries in the form: {query_name: query}
+# Executes each query in order, and returns the resulting contacts, merged and deduped.
+def query_from_list(queries)
+  {}.tap do |contacts|
+    # Reverse sort to start with more recent years.
+    # query_subscribed_contacts dedupes by rejecting duplicate emails.
+    queries.keys.sort.reverse_each do |query_name|
```

Comment:
I realized this `sort.reverse_each` was leftover from an old PR copy/pasted. I'll remove that and just go with `.each |query, query_name|`